Expense/Expense.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
+
+const Expense = new mongoose.Schema({
+    amount: Number,
+    description: String,
+    budget:[{
```

Comment:
Since Expenses are many-to-one relationship with Budgets, this `budget` property _shouldn't_ be an array.